acceptance/openstack/containerinfra/v1/containerinfra.go

```diff
@@ -0,0 +1,83 @@
+package v1
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+	"github.com/gophercloud/gophercloud/openstack/containerinfra/v1/clustertemplates"
+	th "github.com/gophercloud/gophercloud/testhelper"
+)
+
+// CreateClusterTemplate will create a random cluster tempalte. An error will be returned if the
+// cluster-template could not be created.
+func CreateClusterTemplate(t *testing.T, client *gophercloud.ServiceClient) (*clustertemplates.ClusterTemplate, error) {
+	choices, err := clients.AcceptanceTestChoicesFromEnv()
+	if err != nil {
+		return nil, err
+	}
+
+	name := tools.RandomString("TESTACC-", 8)
+	t.Logf("Attempting to create cluster template: %s", name)
+
+	boolFalse := false
+	createOpts := clustertemplates.CreateOpts{
+		Name:                name,
+		MasterFlavorID:      "m1.small",
```

Comment:
Why don't just reuse the choices.FlavorID?

Reply:
That's probably a good idea. This can be updated in a future PR.